### PR TITLE
EPMRPP-38841. Pattern-analysis. View All.

### PR DESCRIPTION
**Pull request by ElenaRomanchuk**
pattern-analysis-view
no-patterns

### CODE REVIEW

File: app/src/pages/common/settingsPage/patternAnalysisTab/patternAnalysisTab.jsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+});
+
+const cx = classNames.bind(styles);
+
+@injectIntl
+@connect(
+  (state) => ({
+    patterns: patternsSelector(state),
+  }),
+  {},
```
Comment (AmsterGet) [inline app/src/pages/common/settingsPage/patternAnalysisTab/patternAnalysisTab.jsx]: You can remove it for now.
Just
```
@connect((state) => ({
  patterns: patternsSelector(state),
}))
```

File: app/src/pages/common/settingsPage/patternAnalysisTab/patternList/patternList.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { PatternListItem } from './patternListItem';
+
+// eslint-disable-next-line react/prefer-stateless-function
```
Comment (AmsterGet) [inline app/src/pages/common/settingsPage/patternAnalysisTab/patternList/patternList.jsx]: Which purpose of it? For future development?

File: app/src/pages/common/settingsPage/patternAnalysisTab/patternList/patternListItem/patternControlPanel/patternControlPanel.jsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+import { InputSwitcher } from 'components/inputs/inputSwitcher';
+import classNames from 'classnames/bind';
+import styles from './patternControlPanel.scss';
+
+const cx = classNames.bind(styles);
+
+@injectIntl
+export class PatternControlPanel extends Component {
+  static propTypes = {
+    intl: intlShape.isRequired,
```
Comment (AmsterGet) [inline app/src/pages/common/settingsPage/patternAnalysisTab/patternList/patternListItem/patternControlPanel/patternControlPanel.jsx]: It seems that this prop is never used.

File: app/src/pages/common/settingsPage/patternAnalysisTab/patternAnalysisTab.jsx (hunk 1 above)
Comment (ElenaRomanchuk) [reply app/src/pages/common/settingsPage/patternAnalysisTab/patternAnalysisTab.jsx]: Fixed

File: app/src/pages/common/settingsPage/patternAnalysisTab/patternList/patternListItem/patternControlPanel/patternControlPanel.jsx (hunk 3 above)
Comment (ElenaRomanchuk) [reply app/src/pages/common/settingsPage/patternAnalysisTab/patternList/patternListItem/patternControlPanel/patternControlPanel.jsx]: Fixed

File: app/src/pages/common/settingsPage/patternAnalysisTab/patternList/patternList.jsx (hunk 2 above)
Comment (ElenaRomanchuk) [reply app/src/pages/common/settingsPage/patternAnalysisTab/patternList/patternList.jsx]: Yes, it was for future usage. But for now I decided extract logic in other component. So, I 've just written this as functional component.

**Pull request merged**